Source/Meadow.Foundation.Libraries_and_Frameworks/Graphics.MicroGraphics/Driver/MicroGraphics.cs

```diff
@@ -848,13 +848,14 @@ public void DrawRectangle(int x, int y, int width, int height, Color color, bool
             }
             else
             {
-                //because we include the starting pixel 
                 width--;
                 height--;
 
+                Console.WriteLine($"DrawRect {x},{y},{width},{height}");
```

Comment:
can you remove this console output, please?

Reply:
yes thx!